akka-javasdk-testkit/src/test/java/akka/javasdk/testkit/eventsourced/CounterEvent.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.testkit.eventsourced;
+
+import com.google.common.collect.Multimap;
```

Comment:
alright, so Jackson knows how to serialize this?

Reply:
no, but that's the point of using sth non-serializable

Reply:
ah 😄